docs/flag_source_configuration.md

```diff
@@ -54,6 +56,8 @@ The relevant `FlagSourceConfigurations` are passed to the operator by setting th
 | DefaultSyncProvider | Defines the default provider to be used, can be set to `kubernetes`, `filepath`, `http` or `flagd-proxy` (experimental).                                                                                                  | optional `string`                                                         | `kubernetes`                 | 
 | RolloutOnChange     | When set to true the operator will trigger a restart of any `Deployments` within the `FlagSourceConfiguration` reconcile loop, updating the injected sidecar with the latest configuration. | optional `boolean`                                                        | `false`                      | 
 | ProbesEnabled       | Enable or disable Liveness and Readiness probes of the flagd sidecar. When enabled, HTTP probes( paths - `/readyz`, `/healthz`) are set with an initial delay of 5 seconds                  | optional `boolean`                                                        | `true`                       |       
+| DebugLogging        | Enable or disable --debug flag of flagd sidecar                                                                                                                                             | optional `boolean`                                                        | `false`                      |
+| RawSidecarArgs      | String arguments passed to the flagd sidecar on startup, flagd documentation can be found [here](https://github.com/open-feature/flagd/blob/main/docs/configuration/configuration.md). Please be aware, flags defined by CRD parameters have bigger value and raw arguments will be ignored. | optional `array of strings`                                               | `""`                         |
```

Comment:
Minor - Regarding the link, shouldn't we target to startup flags document [1] ? :) 

[1] - https://github.com/open-feature/flagd/blob/main/docs/configuration/flagd_start.md

Reply:
good catch, but probably the whole `rawSidecarArgs` will be removed